index.bs

```diff
@@ -282,6 +284,20 @@ interface IntersectionObserver {
 		passed to the {{IntersectionObserver}} constructor.  If no
 		{{IntersectionObserverInit/rootMargin}} was passed to the {{IntersectionObserver}}
 		constructor, the value of this attribute is "0px 0px 0px 0px".
+	: <dfn>scrollMargin</dfn>
+	::
+		Offsets are applied to <a>scrollports</a> on the path from <a>intersection root</a> to <a for="IntersectionObserver">target</a>,
```

Comment:
When this was first discussed with other implementations, we had decided to also apply other forms of clipping, such as `clip-path` ([discussion](https://github.com/w3c/IntersectionObserver/issues/431#issuecomment-1542502858)). Do you think it's best to incorporate that here, or, after prototyping this, do you think it would be best to not do that behavior?

Reply:
Yes, this is already incorporated here.